MaybeGame/EnemyStats.cs

```diff
@@ -0,0 +1,209 @@
+
+            return strength;
+        }
+        public int SetStamina(string enemy)
+        {
+            // Stamina is the ability to sustain prolonged physical or mental effort.
+
+            var stamina = 5;
+
+            switch (enemy)
+            {
+                case "Zombie":
+                    stamina += 2;
+                    break;
+
+                case "Skeleton":
+                    stamina += 2;
+                    break;
+
+                case "Bandit":
+                    stamina += 3;
+                    break;
+
+                case "Theif":
+                    stamina += 2;
+                    break;
+
+                case "Warrior":
+                    stamina += 5;
+                    break;
+
+                case "Berserker":
+                    stamina += 2;
+                    break;
+
+                case "Rat":
+                    stamina += 1;
+                    break;
+            }
+
+            return stamina;
+        }
+        public int SetAgility(string enemy)
+        {
+            // Agility is the ability to move quickly and easily.
+            var agility = 5;
+
+            switch (enemy)
+            {
+                case "Zombie":
+                    agility += 2;
+                    break;
+
+                case "Skeleton":
+                    agility += 2;
+                    break;
+
+                case "Bandit":
+                    agility += 4;
+                    break;
+
+                case "Theif":
+                    agility += 7;
+                    break;
+
+                case "Warrior":
+                    agility += 2;
+                    break;
+
+                case "Berserker":
+                    agility += 5;
+                    break;
+
+                case "Rat":
+                    agility += 5;
+                    break;
+            }
+
+            return agility;
+        }
+        public int SetDexterity(string enemy)
+        {
+            // Dexterity is the readiness and grace in physical activity.
+            var dexterity = 5;
+
+            switch (enemy)
+            {
+                case "Zombie":
+                    dexterity += 2;
+                    break;
+
+                case "Skeleton":
+                    dexterity += 3;
+                    break;
+
+                case "Bandit":
+                    dexterity += 3;
+                    break;
+
+                case "Theif":
+                    dexterity += 5;
+                    break;
+
+                case "Warrior":
+                    dexterity += 4;
+                    break;
+
+                case "Berserker":
+                    dexterity += 1;
+                    break;
+
+                case "Rat":
+                    dexterity += 3;
+                    break;
+            }
+
+            return dexterity;
+        }
+        public int SetHealth(int stamina)
+        {
+            return 50 + (stamina * 3);
```

Comment:
If this is ALL this method does, consider using expression body **=>**

Reply:
Yes.